doc/articles/toc.yml

```diff
@@ -146,6 +146,8 @@
           href: how-uno-works.md
         - name: Best practices
           href: best-practices-uno.md
+        - name: Why use Uno?
```

Comment:
Why use Uno Platform, not just Uno

Reply:
Addressed with https://github.com/unoplatform/uno/pull/15454/commits/d294d19b33b98c7790400978ce0f0febc323c24e